Source/Mod/Menu/OnScreenKeyboardMenu.cs

```diff
@@ -0,0 +1,216 @@
+﻿using System;
+
+namespace Celeste64.Mod;
+
+public class OnScreenKeyboardMenu : Menu
```

Comment:
Probably not a requirement for this PR, but one thing that might be useful is a way to change it to start typing Capital Letters as well. Although, I'm fine with also saving this till later if we'd prefer, unless we want it for save files.

Reply:
Not sure. InputFields may be used for things where sentence-case isnt a requirement or is straight up wrong, though I'd be fine with this behaviour being behind an option passed to the ctor